app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -4012,6 +4015,10 @@ class BrowserTabViewModel @Inject constructor(
         refreshPixelSender.sendCustomTabRefreshPixel()
     }
 
+    fun handleBreakageRefreshPatterns(refreshPatterns: Set<RefreshPattern>) {
```

Comment:
We shouldn't expose this function, as it shouldn't be called from outside the VM. Can you make it private and refactor `whenHandleBreakageRefreshPatternsThenSendBreakageRefreshPixels` so it calls `refreshCta instead`?